packages/composite-checkout/src/components/button.tsx

```diff
@@ -7,7 +7,7 @@ const CallToAction = styled( 'button' )< CallToActionProps >`
 	display: block;
 	width: ${ ( props: CallToActionProps ) => ( props.fullWidth ? '100%' : 'auto' ) };
 	font-size: 16px;
-	border-radius: ${ ( props ) => ( props.buttonType === 'paypal' ? '50px' : '2px' ) };
+	border-radius: 4px;
```

Comment:
We should double-check the PayPal UI after this change.

Reply:
Added to the checklist to verify.

Reply:
Yeah, I kind of liked the consistency of the more square button. Since we're updating the logo, we're still within [PayPal's button guidelines](https://developer.paypal.com/docs/log-in-with-paypal/customize/button-design-guide/). We don't really need to do the color either, but the yellow is pretty identifiable.